package.json

```diff
@@ -29,5 +29,8 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^4.0.0"
+  },
+  "dependencies": {
```

Comment:
ERMM we should probably delete this - 
I think it was when you did `yarn add openai -W` in the root and not just `/server` 

careful - be sure to not run yarn with this but tbh idk should probably be fine